v22.2/create-function.md

```diff
@@ -0,0 +1,110 @@
+---
+title: CREATE FUNCTION
+summary: The CREATE FUNCTION statement creates a user-defined function for a database.
+toc: true
+keywords:
+docs_area: reference.sql
+---
+
+The `CREATE FUNCTION` [statement](sql-statements.html) creates a [user-defined function](user-defined-functions.html) for a schema.
+
+{% include {{ page.version.version }}/misc/schema-change-stmt-note.md %}
+
+## Required privileges
+
+- To define a function, a user must have [`CREATE` privilege](security-reference/authorization.html#supported-privileges) on a schema.
+- To define a function with a [user-defined type](create-type.html), a user must have `USAGE` privilege on a user-defined type.
+- To resolve functions, a user must have at least the `USAGE` privilege on a schema.
+- To call a function, a user must have `EXECUTE` privilege on the function.
+- A user must have privileges on all the objects referenced in the function body at execution time. Privileges on referenced objects can be revoked and later function calls can fail due to lack of permission.
```

Comment:
A user must have privileges on the referenced objects at function definition time as well.

Reply:
Fixed.